src/App.js

```diff
@@ -1,5 +1,43 @@
-function App() {
-  return <div>Hello world</div>
-}
+import { useState } from 'react'
+import { v4 as uuid } from 'uuid'
+
+import ListItem from './01.UI-Elements/ListItem'
+import InputForm from './01.UI-Elements/InputForm'
+
+export default function App() {
+  const [list, setList] = useState({
+    byId: {},
+    allIds: [],
+  })
 
-export default App
+  return (
+    <div className="App">
```

Comment:
You shouldn't need the className anymore.

Reply:
Wasn't sure about it but took it out now. thx